plugins/processors/converter/converter.go

```diff
@@ -345,11 +346,22 @@ func toInteger(v interface{}) (int64, bool) {
 			return 0, true
 		}
 	case string:
-		result, err := strconv.ParseFloat(value, 64)
-		if err != nil {
-			return 0, false
+		if strings.HasPrefix(value, "0x") {
```

Comment:
Instead of this check, just try ParseInt and then fallback to ParseFloat if it fails, this will give us octal support and allow `0XFF` to work as well.  Also, can you add a similar conversion for the unsigned type as well?

Reply:
Hey @danielnelson , thanks for you advices. That's now done (refactoring + string hexa to uint converter). Unfortunately, there is one issue in the CI. CircleCI is OK but Appveyor failed. I may be wrong but I think it's not because of my code (the error is `failed to write dep tree: failed to export golang.org/x/crypto: 
	(1) failed to list versions for https://go.googlesource.com/crypto: fatal: remote error: Internal Server` :thinking: )

Reply:
Sorry about that, just having some issues lately with our builds on appveyor, I'm sure its unrelated.